src/test/unit/common/proxy.test.ts

```diff
@@ -28,7 +29,8 @@ suite('Proxy', () => {
 
     const agent = getHttpsProxyAgent(workspace);
 
-    assert.strictEqual(agent, undefined);
+    // @ts-ignore: cannot test options otherwise
```

Comment:
What does it complaint about?

Reply:
`Property 'proxy' is private and only accessible within class 'HttpsProxyAgent'.`

The ignore seems to be the workaround used in the other tests.